packages/bpk-component-autosuggest/src/BpkAutosuggest.module.scss

```diff
@@ -20,6 +20,8 @@
 
 $bpk-spacing-v2: true;
 
+$arrow-size: bpk-spacing-sm() * 3;
```

Comment:
Value stays the same, only added this as I didn't know the border is the actual arrow and maybe this could make it easier for anyone looking at this in the future